pyannote/database/util.py

```diff
@@ -416,7 +416,7 @@ def load_rttm(file_rttm):
     Parameter
     ---------
     file_rttm : `str`
-        Path to RTTM file.
+        A list of Paths or a Path to RTTM file.
```

Comment:
See my comment above. Let's keep these function simple and only support loading one file at a time...